pkg/sloop/webserver/webfiles/sloop_ui.js

```diff
@@ -514,4 +514,26 @@ function showDetailedTooltip(d, event, parent) {
             }
         });
     }
-}
\ No newline at end of file
+}
+
+$(document).ready(function() {
+    //Set max allowed selected date/time to now
+    const now = new Date();
+    var iso = now.toISOString();
+    var maxDate = iso.substring(0,iso.length-5);
+    elem = document.getElementById("selectedEndTime");
+    elem.value = maxDate;
+    elem.max = maxDate;
+
+    //Display user selected end time on ui after click submit button or refresh the page
+    var userDate = new Date();
+    // check if selected end time happened within 3 seconds
```

Comment:
Why do we value 3 sec here?

Reply:
3 sec is a random reasonable number, any short period is okay